### ISSUE TITLE
false positive, unused variable

### ISSUE THREAD

**Issue by larshp** [abaplint/abaplint#2825]
```abap
DATA textpool TYPE table_of_textpool.
READ TEXTPOOL 'sdf' INTO textpool LANGUAGE sy-langu.
```

@Ennowulff

https://github.com/tricktresor/class_inspector/commit/b9bcf9a0a5a4e9ed8ea5d4ca2fee091c86c57900

![image](https://user-images.githubusercontent.com/5888506/210366861-bda5540d-2d3d-4adf-9c4e-73dd60314a95.png)
